pkg/scheduler/framework/plugins/podtopologyspread/filtering.go

```diff
@@ -309,8 +319,21 @@ func (pl *PodTopologySpread) Filter(ctx context.Context, cycleState *framework.C
 			continue
 		}
 		// judging criteria:
-		// 'existing matching num' + 'if self-match (1 or 0)' - 'global min matching num' <= 'maxSkew'
+		// 'existing matching num' + 'if self-match (1 or 0)' - 'global minimum' <= 'maxSkew'
 		minMatchNum := paths[0].MatchNum
+
```

Comment:
Would it be readable to arrange this to a function like `MinMatchNum()`? It can be a method of `preFilterState`.